mail.pohjoistech.fi.email.json

```diff
@@ -0,0 +1,56 @@
+{
+  "providerId": "mail.pohjoistech.fi",
+  "providerName": "PohjoisMail",
+  "serviceId": "email",
+  "serviceName": "Mail",
+  "version": 1,
+  "syncPubKeyDomain": "domainconnect.pohjoistech.fi",
+  "syncRedirectDomain": "mail.pohjoistech.fi",
+  "logoUrl": "https://r2.pohjoistech.fi/logo.png",
+  "description": "Enables a domain to work with PohjoisMail, a mail service offered by Pohjoistech.",
+  "variableDescription": "ip is the IP address of the server, and key is the DKIM public key.",
+  "records": [
+    {
+      "pointsTo": "%ip%",
+      "type": "A",
+      "host": "pohjoismail",
+      "essential": "OnApply",
+      "ttl": 14400
+    },
+    {
+      "pointsTo": "autodiscover",
+      "type": "CNAME",
+      "host": "pohjoismail",
```

Comment:
This does not make sense, there is A record on the same host. Shouldn't it be host autodiacover pointing to pohjoismail?

Reply:
Ah! I had accidentally mixed up pointsTo and host. Fixed it in the new commit.